idea-plugin/resources/META-INF/plugin.xml

```diff
@@ -0,0 +1,35 @@
+<idea-plugin version="2">
+  <id>com.your.company.unique.plugin.id</id>
+  <name>Plugin display name here</name>
+  <version>1.0</version>
+  <vendor email="support@yourcompany.com" url="http://www.yourcompany.com">YourCompany</vendor>
```

Comment:
Same with this file.  Looks like these files were generated when I was setting up a new IntelliJ install to build this plugin.